src/app/pages/collections/search/landing-page.jsx

```diff
@@ -180,8 +180,31 @@ function ChplLandingPage() {
               <ChplFilterSearchTerm />
               <ChplFilterQuickFilters />
             </Box>
-            <Box pb={4}>
-              <Typography className={classes.subHeaders} align="left" variant="h2" gutterBottom>
+            { editionlessIsOn ? (
```

Comment:
This shouldn't be a ternary. The way this is set up you'll display one thing if the flag is on, and another if it's off, but the way you have it there's nothing that gets displayed when it's off, so this should just be a conditional. Something like this:
```suggestion
            { editionlessIsOn 
            && (
```